modules/highlight/highlight_test.go

```diff
@@ -20,83 +22,83 @@ func TestFile(t *testing.T) {
+				</span></span><span class="line"><span class="cl"><span class="w">	</span>- <span class="l">go test -v -race -coverprofile=coverage.txt -covermode=atomic</span></span></span>
+			`)),
 		},
 		{
 			name:     ".drone.yml - trailing space",
 			numLines: 13,
 			fileName: ".drone.yml",
-			code: `kind: pipeline
-name: default  ` + `
+			code: strings.Replace(strings.TrimSpace(dedent.Dedent(`
+				kind: pipeline
+				name: default
 
-steps:
-- name: test
-	image: golang:1.13
-	environment:
-		GOPROXY: https://goproxy.cn
-	commands:
-	- go get -u
-	- go build -v
-	- go test -v -race -coverprofile=coverage.txt -covermode=atomic
-	`,
-			want: []string{
-				`<span class="line"><span class="cl"><span class="nt">kind</span><span class="p">:</span><span class="w"> </span><span class="l">pipeline</span>`,
-				`</span></span><span class="line"><span class="cl"><span class="w"></span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l">default  </span>`,
-				`</span></span><span class="line"><span class="cl">`,
-				`</span></span><span class="line"><span class="cl"><span class="w"></span><span class="nt">steps</span><span class="p">:</span>`,
-				`</span></span><span class="line"><span class="cl"><span class="w"></span>- <span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l">test</span>`,
-				`</span></span><span class="line"><span class="cl"><span class="w">	</span><span class="nt">image</span><span class="p">:</span><span class="w"> </span><span class="l">golang:1.13</span>`,
-				`</span></span><span class="line"><span class="cl"><span class="w">	</span><span class="nt">environment</span><span class="p">:</span>`,
-				`</span></span><span class="line"><span class="cl"><span class="w"></span><span class="w">		</span><span class="nt">GOPROXY</span><span class="p">:</span><span class="w"> </span><span class="l">https://goproxy.cn</span>`,
-				`</span></span><span class="line"><span class="cl"><span class="w">	</span><span class="nt">commands</span><span class="p">:</span>`,
-				`</span></span><span class="line"><span class="cl"><span class="w"></span><span class="w">	</span>- <span class="l">go get -u</span>`,
-				`</span></span><span class="line"><span class="cl"><span class="w">	</span>- <span class="l">go build -v</span>`,
-				`</span></span><span class="line"><span class="cl"><span class="w">	</span>- <span class="l">go test -v -race -coverprofile=coverage.txt -covermode=atomic</span>`,
-				`</span></span><span class="line"><span class="cl"><span class="w">	</span></span></span>`,
-			},
+				steps:
+				- name: test
+					image: golang:1.13
+					environment:
+						GOPROXY: https://goproxy.cn
+					commands:
+					- go get -u
+					- go build -v
+					- go test -v -race -coverprofile=coverage.txt -covermode=atomic
+			`))+"\n", "name: default", "name: default  ", 1),
```

Comment:
Oh, here, why the `Replace(..., "name: default", "name: default  ", 1)` is needed. The spaces should have been trimmed?

Reply:
The test case previously also specifically tested for these trailing whitespaces and our editors are configured to trim trailing whitespace, that's why I did it like this. Previous test also never had literal lines ending in trailing whitespace because of it.